components/Article/InlineCode/index.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import InlineCode from './index';
+import type { FC } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+type InlineCodeArgs = {
```

Comment:
This typing feels weird. Please check the CONTRIBUTING guidelines (there should be a Meta type)